index.js

```diff
@@ -10,4 +10,8 @@ https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
 
 "use strict";
 
+var fluid = fluid || require("infusion");
+var gpii = fluid.registerNamespace("gpii");
 require("./src/js/firstDiscoveryServer.js");
+
+gpii.firstDiscovery.server();
```

Comment:
I have done a lot of work in recent text fixtures to propagate the port into sensible base URLs, it's certainly possible to move some of this into the defaults for gpii.express.